doc/installation.rst

```diff
@@ -76,176 +91,183 @@ On macOS, open the disk image and drag Spyder to your :guilabel:`Applications` f
+You can then run Spyder by the same methods :ref:`as with Anaconda <install-anaconda-running>`, except make sure to launch the start menu shortcut with ``(spyder-env)`` in the name, select the ``spyder-env`` environment on the left before launching it with Navigator, or type ``conda activate spyder-env`` before launching it on the command line.
 
-.. _WinPython: https://winpython.github.io/
+See :ref:`our FAQ question <using-existing-environment>` for more information about how to use Spyder with your existing Conda environments.
 
 
-macOS
-~~~~~
+.. _install-conda-update:
 
-Thanks to the `MacPorts project`_, Spyder can be installed using its ``port`` package manager; however, the included Spyder version may be out of date or have MacPorts-specific issues outside of Spyder's control.
+Updating with Conda
+-------------------
 
-.. _MacPorts project: https://www.macports.org/
+With any Conda-based distribution and Spyder installed in its own environment (recommended), update Conda itself, active the environment then update Spyder.
+In your system terminal (or Anaconda Prompt if on Windows), run:
 
-There are `several versions`_ available from which you can choose.
+.. code-block:: shell
 
-.. _several versions: https://ports.macports.org/search/?q=spyder&name=on
+   conda update -n base conda
+   conda activate spyder-env
+   conda update spyder
 
+In case you get an error trying to update, just remove the existing environment (if using one other than ``base``):
 
-Linux
-~~~~~
+.. code-block:: shell
 
-.. warning::
+   conda remove -n spyder-env --all
 
-   Distribution packages are created by third parties, are often outdated relative to the current Spyder release, and may contain bugs and be missing features relative to the current supported version.
-   As such, given we are not able to provide official support for them, we strongly recommend using :ref:`anaconda_install_ref` on Linux whenever practical (or :ref:`pip<pip_install_spyder_ref>`, for advanced users).
+And then :ref:`recreate a fresh one <install-conda-installing>`.
 
-Spyder can be installed via third-party distro packages on most common Linux distributions.
 
-Running Spyder installed this way will generally be the same as any other distro-installed application.
-Alternatively, it can be launched from the terminal with ``spyder`` (or ``spyder3``, on older versions of some distros).
+.. _install-anaconda-base:
 
+Anaconda base
+~~~~~~~~~~~~~
 
-Ubuntu
-------
+While we recommend always using a dedicated environment, with Anaconda you can also run the bundled copy of Spyder from the built-in ``base`` environment.
 
-Using the package manager:
+.. caution::
 
-.. code-block:: bash
+   The bundled Spyder version can often be quite out of date, missing new features and bug fixes from the latest version, and if you install, change or remove other packages, there is a chance of dependency conflicts or a broken Spyder installation.
```

Comment:
```suggestion
   The bundled Spyder version can often be quite out of date, missing new features and bug fixes from the latest version, and if you install, change or remove other packages, there is an important chance of dependency conflicts or a broken Spyder installation.
```

To discourage people even more from using Spyder from the base env,

Reply:
"an important chance" doesn't make any semantic sense here in English. However, we can use a more appropriate word for emphasis:

```suggestion
   The bundled Spyder version can often be quite out of date, missing new features and bug fixes from the latest version, and if you install, change or remove other packages, there is a significant chance of dependency conflicts or a broken Spyder installation.
```